source/develop/release-management/features/storage/pass-discard-from-guest-to-underlying-storage.html.md

```diff
@@ -0,0 +1,235 @@
+---
+title: PassDiscardFromGuestToUnderlyingStorage
+authors: ishaby
+wiki_title: Features/PassDiscardFromGuestToUnderlyingStorage
+feature_name: Pass discard from guest to underlying storage
+feature_modules: engine/vdsm/api-model
+feature_status: Work in progress for oVirt 4.1
+---
+
+# Pass discard from guest to underlying storage
+
+## Summary
+
+### How thinly provisioned disks are used today
+Today, when the engine creates a thinly provisioned disk on a block domain, a logical volume with a preconfigured size is created, regardless its virtual size.<br/>
+Each time that vdsm reaches the current lv's watermark, it extends the lv, so that the disk grows again and again until it reaches its virtual size.<br/>
```

Comment:
VDSM doesn't reach anything. When the watermark comes near the LV's size, the SPM extends the LV.